libs/application/types/src/lib/Fields.ts

```diff
@@ -826,6 +828,39 @@ export interface DisplayField extends BaseField {
   value: (answers: FormValue) => string
 }
 
+type KeyValueItem = {
+  width?: 'full' | 'half' | 'snug'
+  keyText?: FormText
+  valueText?: FormText
+  boldValueText?: boolean
+  lineAboveKeyText?: boolean
+}
+
+type AttachmentItem = {
+  width?: 'full' | 'half'
+  fileName: FormText
+  fileSize?: FormText
+  fileType?: FormText
+}
+
+export interface OverviewField extends BaseField {
+  readonly type: FieldTypes.OVERVIEW
+  component: FieldComponents.OVERVIEW
+  title: FormText
+  description?: FormText
+  backId?: string
+  fieldType: 'keyValue' | 'attachments'
```

Comment:
Maybe we should add "table" as well?

Reply:
I don't remember a specific application but it there are some. It's useful when displaying data from tablerepeater f.x.